run/send-push-message

```diff
@@ -37,6 +37,17 @@ function send_sns () {
   python3 /root/bin/send_sns.py -t "$AWS_SNS_TOPIC_ARN" -s "$title" -m "$message"
 }
 
+function send_matrix() {
+  log "Send Matrix message."
+
+  if [ -z ${HOME+x} ]
+  then
+      export HOME=/root
+  fi
```

Comment:
is this needed? (it was for matrixcli which directly used it, but your new code doesn't)

Reply:
No, you are right: it's obsolete - removed the four lines, still works as expected.